### DISCUSSION TITLE
Help with MacOS 10.0.0 public beta, 10.0, and 10.1

### DISCUSSION THREAD

**Discussion by johnkgibson** [utmapp/UTM#3246] [UTM Help (macOS)] [answered]
I have the VM working but I can't figure out how to get the mouse working when the mouse is captured. I looked at the QEMU pages to try to figure out but translating it over to UTM is frustrating. 

I successfully installed 9.2.2 and Mac OS X 10.3 and that VM is working really well.  I used the public beta    when it released when I was the IT systems admin for an ad agency. I had to figure out when to make the switch to OS 10. 

Any help would be very appreciated.

**Comment by johnkgibson** [comment]
I finally found the answer.  If anyone is interested I can write up the steps to replicate.

**Reply by hrishter** [reply]
Hey mate, how did you fix this issue? I've tried a few ways and nothing seems to have worked. Cheers

**Reply by johnkgibson** [reply]
Hey, 
Sorry, I didn't get back to you. I do have a good excuse. In February of 2023, I was walking home, crossed a street, and was hit by a car. So, I have been in recovery. I will see if I can find my notes.

**Reply by johnkgibson** [reply, marked as answer]
Sorry for taking so long - Part of a TBI is memory issues, and I didn't remember this until a couple of days ago

The fix is changing the machine argument to via=cuda

<img width="818" alt="Screenshot 2025-06-27 at 11 15 20 AM" src="https://github.com/user-attachments/assets/6992ae8d-840d-4d17-865c-5f4a9b142959" />


**Here is the configuration file for the public beta**

[OS X 10 Public Beta (config only).utm.zip](https://github.com/user-attachments/files/20961655/OS.X.10.Public.Beta.config.only.utm.zip)

Here is a repository of various configurations that were created by adespoton. 
[https://github.com/adespoton/utmconfigs](url)

**Reply by OsmoticDuck** [reply]
OMG, it's working!!!! Very very appreciated for your answer!!!

**Reply by SoCuul** [reply]
This should definitely be marked as the default in the Classic macOS UTM config!

**Comment by hrishter** [comment]
Oh mate I'm really sorry to hear that. I hope your recovery has been going
well and there isn't too much lasting damage, and most importantly, I hope
you're still able to keep a positive look on things especially when rehab
and recovery isn't moving as well as you may hope. I know I'm just a
stranger on the internet, but my best to you and your loved ones.

**Reply by johnkgibson** [reply]
I’m doing pretty well. My doctors have urged me to create a local support group for traumatic brain injury survivors. It’s been a rough 16 months. 

I’ve been using the time to do small projects that don’t require computers, like plumbing, yard work, and recycling things accumulated over my thirty-six years of marriage with five kids. 

I’m building a workshop, which I’ll use to make cool stuff. I turned sixty last year before the accident and decided to embrace my inner weirdness. 

I’m not abandoning computers. I worked in IT since 1985, and they are a big love of mine. 

I’m back doing work on some emulation work with UTM and QEMU and building some “prebuilt” VMs. 

I picked up a couple of ancient Macs and replaced their internal hardware with a Raspi because the idea makes me giddy. 

Since the temperature will be F 96°/ C 35° I’m going to stay inside so I’ll be searching my backup systems to find that Virtual Machine you asked about. If I locate it, I’ll compress it and make it available to you, and anyone else interested. 

Have an awesome day

**Reply by andyfrueh** [reply]
FWIW, I had to re-enable USB as Macs of this vintage dropped ADB and never (AFAIK) had PS/2 ports.  Once I re-enabled USB I could click the"capture mouse" button in the VM Window's toolbar and it grabbed the mouse.

**Comment by djmaximum17** [comment]
when I did that, it panics on boot
<img width="1012" height="764" alt="Screenshot 2026-02-20 at 11 05 51 PM" src="https://github.com/user-attachments/assets/073fb935-c411-4ecb-8acf-d129ea0a54e2" />
<img width="912" height="750" alt="Screenshot 2026-02-20 at 11 04 08 PM" src="https://github.com/user-attachments/assets/cac8fc6b-3c4c-4b6b-b85c-ac8101699386" />
